MovieApp/MovieApp/App/Features/MovieBrowser/Components/MovieCellHPv1/MovieCell.swift

```diff
@@ -1,34 +1,35 @@
 import UIKit
 import Kingfisher
 
-class MovieCell: UICollectionViewCell {
+class MovieCell: UITableViewCell {
     
     static let reuseIdentifier = String(describing: MovieCell.self)
     
-    let descriptionFontSize: CGFloat = 14
+    var movieId: Int!
+    
```

Comment:
no need for this

Reply:
[Removed](https://github.com/lihter/Movie-App/pull/41/commits/3383b62f0e45eb7e1f5ba5b650a01beaa73387a1#diff-59acdf9fea7a34d1960015778fa4fae8eba6ce40a9b91286052e7d63a97de079L8)